animated-gif/src/main/jni/gifimage/gif.cpp

```diff
@@ -1147,6 +1147,39 @@ jint GifFrame_nativeGetDurationMs(JNIEnv* pEnv, jobject thiz) {
   return spNativeContext->durationMs;
 }
 
+/**
+ * Gets the color (as an int, as in Android) of the transparent pixel of this frame
+ *
+ * @return the color (as an int, as in Android) of the transparent pixel of this frame
+ */
+jint GifFrame_nativeGetTransparentPixel(JNIEnv* pEnv, jobject thiz) {
+  auto spNativeContext = getGifFrameNativeContext(pEnv, thiz);
+
+  ColorMapObject* pColorMap = spNativeContext->spGifWrapper->get()->SColorMap;
```

Comment:
I think you have to check if we have a local or global color table. That's the code used for getting the table when we render the frame:
```
  // Get the right color table to use.
  ColorMapObject* pColorMap = spNativeContext->spGifWrapper->get()->SColorMap;
  SavedImage* pSavedImage = &pGifWrapper->get()->SavedImages[frameNum];
  if (pSavedImage->ImageDesc.ColorMap != NULL) {
    // use local color table
    pColorMap = pSavedImage->ImageDesc.ColorMap;
    if (pColorMap->ColorCount != (1 << pColorMap->BitsPerPixel)) {
      pColorMap = sDefaultColorMap;
    }
  }
```